make/mt-os/mt-freertos-linux.mk

```diff
@@ -16,7 +16,7 @@
 CC ?= gcc
 AR ?= ar
 
-IOTC_COMMON_COMPILER_FLAGS += -fPIC
+IOTC_COMMON_COMPILER_FLAGS += -fPIC -DIOTC_TARGET_FREERTOS=1
```

Comment:
`IOTC_TARGET_FREERTOS` probably this isn't required since there is already a specific for FreeRTOS: `IOTC_PLATFORM_IS_FREERTOS `

Reply:
Please see my note above.

Reply:
True, ok, thx for the explanation.